src/discussions/learners/learner/LearnerFooter.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Icon, OverlayTrigger, Tooltip,
+} from '@edx/paragon';
+import {
+  Edit, Error,
+  QuestionAnswer,
+} from '@edx/paragon/icons';
+
+import messages from './messages';
+import { learnerShape } from './proptypes';
+
+function LearnerFooter({
+  learner,
+  intl,
+}) {
+  return (
+    <div className="d-flex align-items-center">
+      <Icon src={QuestionAnswer} className="mx-2 my-0" />
+      <span style={{ minWidth: '2rem' }}>
+        {learner.replies}
+      </span>
+      <Icon src={Edit} className="mx-2 my-0" />
+      <span style={{ minWidth: '2rem' }}>
+        {learner.threads}
+      </span>
+      {learner.active_flags
+        && (
+          <OverlayTrigger
+            overlay={(
+              <Tooltip>
+                {intl.formatMessage(messages.reported, { reported: learner.active_flags })}
+                {learner.inactive_flags
+                  && (
+                    <>
+                      <br />
+                      {intl.formatMessage(messages.previouslyReported, { previously_reported: learner.inactive_flags })}
+                    </>
+                  )}
+              </Tooltip>
```

Comment:
You can use a flex div to style this rathe than adding a br. That will also allow left-aligning this as shown n the mockup.